openedx/core/djangoapps/content/learning_sequences/services.py

```diff
@@ -0,0 +1,18 @@
+"""
+Learning Sequences Runtime Service
+"""
+
+
+from .api.outlines import get_user_course_outline_details
```

Comment:
Please import from `.api` directly. That's the only stable place we make guarantees about (we echo out those exports in `api/__init__.py`.